bot/exts/help_channels/_cog.py

```diff
@@ -410,15 +418,20 @@ async def move_to_in_use(self, channel: discord.TextChannel) -> None:
             category_id=constants.Categories.help_in_use,
         )
 
-        timeout = constants.HelpChannels.idle_minutes * 60
+        timeout = constants.HelpChannels.idle_minutes_others * 60
 
         log.trace(f"Scheduling #{channel} ({channel.id}) to become dormant in {timeout} sec.")
         self.scheduler.schedule_later(timeout, channel.id, self.move_idle_channel(channel))
         _stats.report_counts()
 
     @commands.Cog.listener()
     async def on_message(self, message: discord.Message) -> None:
-        """Move an available channel to the In Use category and replace it with a dormant one."""
+        """
+        Move an available channel to the In Use category and replace it with a dormant one.
+
+        Update the `last_message_times` cache based on the current timestamp. If the message
+        author is the claimant of this channel, also update the `claimant_last_message_times` cache.
```

Comment:
Do you think this should be removed now that the code for this has been moved elsewhere? `on_message` still technically causes this to happen, but it's no longer directly responsible for it.